.dockerignore

```diff
@@ -1,2 +1,10 @@
 .virtualenv
 .venv
+.mypy_cache
+.pytest_cache
+.git
+build
+dist
+images
+__pycashe__
+vendor
```

Comment:
Vendor is go-specific if I'm not mistaken. We can remove that.

Reply:
If we build an image for python only code - it makes sense to filter out all go-specific code.

Reply:
Do you mean that we should remove it from git repository instead? 
https://github.com/danihodovic/celery-exporter/tree/master/vendor
I think go is used for celery-mixin

Reply:
Ah ok